modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -149,11 +149,13 @@ public AbstractTypeScriptClientCodegen() {
         typeMapping.put("DateTime", "Date");
         typeMapping.put("binary", "any");
         typeMapping.put("File", "any");
+        typeMapping.put("file", "any");
```

Comment:
This is a mapping for the OAS2 built-in `file` type. It's similar to the existing mapping defined in `TypeScriptAngularClientCodegen.java`:
```
typeMapping.put("file", "Blob");
```
Without that, the updated `toModelName()` logic would add a bogus import for api endpoints that deal with type:file (`imageUpload` endpoint in PetStore)

Frankly, I'm not sure if the old mapping in line 151 here is needed at all.